packages/core/stories/patterns/keyboard-shortcuts/keyboard-shortcuts.stories.tsx

```diff
@@ -0,0 +1,325 @@
+  useHotkeys(
+    "meta+home",
+    (e) => {
+      e.preventDefault();
+      alert("Top of list triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+u",
+    (e) => {
+      e.preventDefault();
+      alert("Test shortcut triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+y",
+    (e) => {
+      e.preventDefault();
+      alert("Test shortcut triggered");
+    },
+    { enabled: shortcutsEnabled },
+  );
+  useHotkeys(
+    "meta+shift+k",
+    (event) => {
+      event.preventDefault();
+      setFilter("");
+      setOpen(true);
+    },
+    { enabled: shortcutsEnabled },
+  );
+
+  const filteredShortcuts: Shortcut[] = shortcutList.filter((s) =>
+    s.label.toLowerCase().includes(filter.trim().toLowerCase()),
+  );
+
+  const handleDialogOpen = (): void => {
+    setFilter("");
+    setOpen(true);
+  };
+  const handleDialogChange = (value: boolean): void => {
+    setOpen(value);
+    if (value) setFilter("");
+  };
+  const handleSwitchChange = (event: ChangeEvent<HTMLInputElement>): void =>
+    setShortcutsEnabled(event.target.checked);
+  const handleFilterChange = (event: ChangeEvent<HTMLInputElement>): void =>
+    setFilter(event.target.value);
+
+  return (
+    <HotkeysProvider>
+      <StackLayout gap={1}>
+        <Button data-testid="dialog-button" onClick={handleDialogOpen}>
+          Keyboard shortcuts panel
+        </Button>
+        <FlexLayout align="center" gap={1}>
+          <Text>hit </Text>
+          <FlexLayout align="center" gap={0}>
+            <Kbd>meta</Kbd>+<Kbd>shift</Kbd>+<Kbd>K</Kbd>
+          </FlexLayout>
+          <Text>to open the keyboard shortcuts panel </Text>
+        </FlexLayout>
+      </StackLayout>
+      <Dialog
+        open={open}
+        onOpenChange={handleDialogChange}
+        id="keyboard-shortcuts-dialog"
+        size="medium"
+        className="keyboardShortcuts-dialog"
+      >
+        <DialogContent>
+          <StackLayout gap={3}>
+            <FlexLayout gap={1}>
+              <Switch
+                checked={shortcutsEnabled}
+                onChange={handleSwitchChange}
+              />
+              <FlexLayout className="keyboardShortcuts-description">
+                <Text>Turn on keyboard shortcuts</Text>
+              </FlexLayout>
+            </FlexLayout>
+            {shortcutsEnabled && (
+              <StackLayout gap={1}>
+                <Text className="keyboardShortcuts-actions-title" styleAs="h3">
+                  Actions
+                </Text>
+                <StackLayout gap={filteredShortcuts.length ? 3 : 0.75}>
+                  <Input
+                    onChange={handleFilterChange}
+                    value={filter}
+                    bordered
+                    variant="secondary"
+                    placeholder="Filter actions"
+                    startAdornment={<FilterIcon color="secondary" />}
+                    aria-label="Filter actions"
```

Comment:
```suggestion
                    inputProps={{ "aria-label": "Filter actions" }}
```
I think we should use `inputProps` here so the `aria-label` is applied to the actual `<input>` element (not the outer wrapper `<div>`). Thoughts?